Dockerfile

```diff
@@ -61,6 +61,10 @@ COPY tests /app/tests
 ENV PYTHONPATH=.
 COPY bin/startup.sh /app/startup.sh
 COPY redis.conf /app/redis.conf
+
+# Set the SWEEP_VERSION environment variable to the current date and time during image build
+ARG SWEEP_VERSION
+ENV SWEEP_VERSION=${SWEEP_VERSION:-$(date +%Y%m%d%H%M)}
```

Comment:
#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 2.03kB done
#1 DONE 0.0s
Dockerfile:67
--------------------
  65 |     # Set the SWEEP_VERSION environment variable to the current date and time during image build
  66 |     ARG SWEEP_VERSION
  67 | >>> ENV SWEEP_VERSION=${SWEEP_VERSION:-$(date +%Y%m%d%H%M)}
  68 |     RUN chmod u+x /app/startup.sh
  69 |     
--------------------
ERROR: failed to solve: Syntax error - can't find = in "+%Y%m%d%H%M)}". Must be of the form: name=value
Error: buildx failed with: ERROR: failed to solve: Syntax error - can't find = in "+%Y%m%d%H%M)}". Must be of the form: name=value